src/ResourceManagement/EventHub/EventHub.Tests/TestHelper/EventHubManagementHelper.cs

```diff
@@ -0,0 +1,151 @@
+﻿//  
+// Copyright (c) Microsoft.  All rights reserved.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+namespace EventHub.Tests.TestHelper
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Management.EventHub;
+    using Microsoft.Azure.Management.EventHub.Models;
+    using Microsoft.Azure.Management.Resources;
+    using Microsoft.Azure.Management.Resources.Models;
+    using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+    using Newtonsoft.Json;
+    using System.Security.Cryptography;
+    using System.Threading.Tasks;
+    using Newtonsoft.Json.Serialization;
+    using Newtonsoft.Json.Converters;
+
+    public static class EventHubManagementHelper
+    {
+        internal const string DefaultLocation = "West US"; 
```

Comment:
Can you give me an example of a service using it? We do not have that API in our client.